.github/workflows/python-package.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       matrix:
         os: [macos-latest, ubuntu-latest]
-        python-version: [3.5, 3.6, 3.7, 3.8]
+        python-version: [3.6, 3.7, 3.8]
```

Comment:
CI fails for Python 3.5 because of some Pathlib issue. Not sure how to resolve it, so simply remove Python 3.5.